src/controllers/controller.radar.js

```diff
@@ -65,12 +67,17 @@ module.exports = function(Chart) {
 			meta.dataset.pivot();
 
 			// Update Points
-			helpers.each(points, function(point, index) {
-				me.updateElement(point, index, reset);
-			}, me);
+			for (i = 0, ilen = points.length; i < ilen; ++i) {
```

Comment:
`++i` is a little non-standard. The file has `i++` everywhere else, which I see much more commonly

Reply:
I'm using `++i` in for loops but it's an habit that comes from C/C++ which is (was?) an optimization. I'm not sure it has any incidence in JavaScript though.

Reply:
v8 produces the same bytecode in either case

Reply:
The habit for me came from my c++ days too. Will try and address the comments tomorrow